app/main.py

```diff
@@ -1,3 +1,66 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: object) -> object:
```

Comment:
Add better annotation for `other`. `Union[Distance, int]` or `Distance | int` if you're using python 3.10 suits better. Fix in other places
The same for return annotation - it should be `Distance` class